### ISSUE TITLE
[one-cmds] Deprecate one-build

### ISSUE THREAD

**Issue by mhs4670go** [Samsung/ONE#10345]
## What

Let's deprecate _one-build_.

## Why

AFAIK, there's no place to use _one-build_ script.

So, it's time to remove this script and related test from `one-cmds`.

**Comment by seanshpark** [comment]
As talked off-line, please check internal CI repos :)

**Comment by mhs4670go** [comment]
I searched all the codes at internal repo that contains `one-build` and seems okay to deprecate `one-build` from `one-cmds`. 

@seanshpark What's your take on this?

**Comment by seanshpark** [comment]
> What's your take on this?

Lets' go :)

@lemmaa, any comments?

**Comment by seanshpark** [comment]
1/ we can remove one-build
2/ we can display a warning message for a while and then remove one-build

I think 2 is what happens for most tools

**Comment by mhs4670go** [comment]
> 1/ we can remove one-build 2/ we can display a warning message for a while and then remove one-build
> 
> I think 2 is what happens for most tools

Yeah, actually, as of now we are displaying a warning if the configuration file has a `one-build` section. For 2, as a first step, we can just turn off the `one-build` test in the `one-cmds` test.

**Comment by lemmaa** [comment]
> > What's your take on this?
> 
> Lets' go :)
> 
> @lemmaa, any comments?

Let's go!

**Comment by seanshpark** [comment]
we can deprecate `one-build` this year (2025)

**Comment by seanshpark** [comment]
I'll start this from next week.

**Comment by seanshpark** [comment]
done
